baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/IncubatingMethodTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.errorprone;
+
+import com.google.errorprone.CompilationTestHelper;
+import org.junit.Test;
+
+public class IncubatingMethodTest {
+
+    // We define our own simple conjure incubating annotation at the right path:
+    private static final String[] ANNOTATION_DEFINITION =
+            new String[] {"package com.palantir.conjure.java.lib.internal;", "public @interface Incubating {}"};
```

Comment:
Might be simpler to add a `testCompile` dependency on `conjure-lib` so we don't have to define this in tests, and we know we're using the right fqcn

Reply:
Sure, that makes sense. Had to remember how to use `version.props` correctly...